x-pack/test/functional/apps/canvas/lens.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function canvasLensTest({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['canvas', 'common', 'header', 'lens']);
+  const esArchiver = getService('esArchiver');
+
+  describe('lens in canvas', function () {
+    this.tags('skipFirefox');
```

Comment:
skip firefox?

Reply:
Several of our canvas tests skip firefox due to some weird behavior where the browser refuses to detect elements that actually exist and causes flakiness

Reply:
Yeah, I just copied it over from there. I will give it a try and see whether everything works for this specific test.